docs/source/modules/zos_lineinfile.rst

```diff
@@ -33,7 +33,7 @@ src
 
   The USS file must be an absolute pathname.
 
-  Generation data set (GDS) relative name of generation already created. ``e.g. SOME.CREATION(-1.)``
+  Generation data set (GDS) relative name of generation already created. e.g. *SOME.CREATION(-1*).
```

Comment:
it seems to me like the '*' should be placed outside the close paren ')'
> e.g. *SOME.CREATION(-1*).

Reply:
Does seem like that but the doc decided to generate it this way and I reviewed it, see below 
<img width="1078" alt="image" src="https://github.com/user-attachments/assets/700b6f69-5127-4736-a1bd-c2ff15e533e5">